third_party/icu/source/i18n/plurrule.cpp

```diff
@@ -12,24 +12,16 @@
 #if defined(STARBOARD)
 #include "starboard/client_porting/cwrappers/pow_wrapper.h"
 #include "starboard/client_porting/poem/assert_poem.h"
-#include "starboard/client_porting/poem/string_poem.h"
 #endif  // defined(STARBOARD)
 #include <math.h>
-#if !defined(STARBOARD)
 #include <stdio.h>
-#endif
 
 #include "unicode/utypes.h"
 #include "unicode/localpointer.h"
 #include "unicode/plurrule.h"
 
 // These files must be included after plurrule.h, for the #define in the poems
 // to work successfully.
```

Comment:
Maybe remove the empty line and comments we've inserted on lines 22–24.
https://chromium.googlesource.com/chromium/deps/icu/+/d1e2bd28/source/i18n/plurrule.cpp#17